spec/index.html

```diff
@@ -533,6 +540,309 @@ <h3>Overview</h3>
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in shared hashes">
+          <!--
+            :p :q _:b0 .
+            :p :q _:b1 .
+            _:b0 :p _:b2 .
+            _:b1 :p _:b3 .
+            _:b2 :r _:b3 .
+          -->
+        </pre>
+
+        <p><a href="#ca-2">Step 2</a> is called four times, with each blank node
+          (`_:b0`, `_:b1`, `_:b2`, and `_:b3`)
+          to populate <a>blank node to quads map</a>:</p>
+
+        <table id="ex-ca-bn-quads-map-shared"
+               class="hash-related numbered">
+          <caption>Blank node to quads map for shared hashes</caption>
+          <thead><tr>
+            <th><var>label</var></th>
+            <th><var>Q</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`_:b0`</td>
+              <td>`:p :q _:b0 .`<br/>``_:b0 :p _:b2 .`</td>
+            </tr>
+            <tr>
+              <td>`_:b1`</td>
+              <td>`:p :q _:b1 .`<br/>`_:b1 :p _:b3 .`</td>
+            </tr>
+            <tr>
+              <td>`_:b2`</td>
+              <td>`_:b0 :p _:b2 .`<br/>`_:b2 :r _:b3 .`</td>
+            </tr>
+            <tr>
+              <td>`_:b3`</td>
+              <td>`_:b1 :p _:b3 .`<br/>`_:b2 :r _:b3 .`</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p><a href="#ca-3">Step 3</a> generates the first-degree
+          hash for each blank node, which is explored further
+          in <a href="#ex-shared-hashes"></a>
+          (note that the hashes for `_:b0` and `_:b1` are shared):</p>
+
+        <table id="ex-ca-1d-hashes-shared"
+               class="hash-related numbered">
+          <caption>Hash to blank nodes map shared hashes</caption>
+          <thead><tr>
+            <th><var>hash</var></th>
+            <th><var>label(s)</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`3b26142829b8887d011d779079a243bd61ab53c3990d550320a17b59ade6ba36`</td>
+              <td>`_:b0`, `_:b1`</td>
+            </tr>
+            <tr>
+              <td>`15973d39de079913dac841ac4fa8c4781c0febfba5e83e5c6e250869587f8659`</td>
+              <td>`_:b2`</td>
+            </tr>
+            <tr>
+              <td>`7e790a99273eed1dc57e43205d37ce232252c85b26ca4a6ff74ff3b5aea7bccd`</td>
+              <td>`_:b3`</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p><a href="#ca-4">Step 4</a> creates canonical identifiers for each
+          blank node which has a unique hash:</p>
+
+        <table id="ex-ca-1d-hashes"
+               class="hash-related numbered">
+          <caption>First degree hashes for shared hashes</caption>
+          <thead><tr>
+            <th><var>label</var></th>
+            <th><var>canonical identifier</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`_:b2`</td>
+              <td>`_:c14n1`</td>
```

Comment:
Thanks, we'll need to go through and double check all the values. I think I'll instrument my implementation to output information that can be fairly easily turned into these examples. It will be necessary if/when we do more complicated examples, as doing this stuff by hand is quite challenging.